### DISCUSSION TITLE
Windows11 上转化精度失败

### DISCUSSION THREAD

**Discussion by wusskk** [LearningInfiniTensor/.github#1] [Q&A] [answered]
1. 部署环境
- rust版本
rustc 1.79.0 (129f3b996 2024-06-10)
- rust toolchain
stable-x86_64-pc-windows-gnu (default)
- 系统
Windows11，非WSL

2. 问题详情
执行精度转化时报错，使用命令` cargo cast --model G:\rust\LLM\models\TinyLlama-1.1B-Chat-v1.0\ --dt f16`
报错截图：
![image](https://github.com/LearningInfiniTensor/.github/assets/78784599/4ed5435f-42c1-4d0f-a75d-2b04c7ccefc9)
手动添加打印target后发现target参数无输入时自动生成的路径存在问题，截图如下：
![image](https://github.com/LearningInfiniTensor/.github/assets/78784599/ddfbad36-bf5d-4bb5-946b-dac4bea1cc4c)

![image](https://github.com/LearningInfiniTensor/.github/assets/78784599/91386791-d4ca-41fa-9c34-28688d73398c)

3. 解决方法（**待确认**？不确定该方法是否存在问题）
- 输入命令时直接添加target选项： `cargo cast --model "G:\rust\LLM\models\TinyLlama-1.1B-Chat-v1.0" --dt f16 --target "G:\rust\LLM\models\TinyLlama-1.1B-Chat-v1.0_F16"`

4. **新问题**
对话时崩溃，截图如下：
![image](https://github.com/LearningInfiniTensor/.github/assets/78784599/b92731da-cced-47ba-bd7a-1fff933b6367)

怀疑与之前的问题有关？但是太晚了，明天再说。

**Comment by YdrMaster** [comment, marked as answer]
模型下载问题，看[新的手册](https://github.com/LearningInfiniTensor/.github/blob/main/InfiniLM-user-guide/doc.md#3-%E4%B8%8B%E8%BD%BD%E6%A8%A1%E5%9E%8B)重新下载模型即可
